examples/language/lib/functions/getters_setters.dart

```diff
@@ -0,0 +1,42 @@
+/// Defines a private variable secret with `_`.
+String _secret = 'Hello';
+
+/// The public top-level getter.
+/// It provides read access to the [_secret].
+String get secret {
+  print('Getter was used!');
+  return _secret.toUpperCase();
+}
+
+/// The public top-level setter.
+/// It provides write access to the [_secret].
+set secret(String newMessage) {
+  print('Setter was used! New secret: "$newMessage"');
+  if (newMessage.isNotEmpty) {
+    _secret = newMessage;
+  }
```

Comment:
Good example! However, should the "New secret: ..." portion be printed out separately in the if statement? As otherwise it's not the new secret.

```
  print('Setter was used!');
  if (newMessage.isNotEmpty) {
    _secret = newMessage;
    print('New secret: "$newMessage"');
  }
```